content/running-amazon-ec2-workloads-at-scale/before/self_paced/_index.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "...On your own"
+chapter = true
+weight = 5
++++
+
+### Running the workshop self-paced, in your own AWS account
+
+\
+To complete this workshop, have access to an AWS account with administrative permissions. An IAM user with administrator access (**arn:aws:iam::aws:policy/AdministratorAccess**) will do nicely.
+
+You can start the workshop by heading to [**Setup with CloudFormation**](/running-amazon-ec2-workloads-at-scale/launch_cloudformation.html)
```

Comment:
The only thing that I was thinking after reading the changes is perhaps as a very optional thing, highlight here as well the importance of cleaning up the workshop resources once completed so there are no surprises if they leave the workshop half way before completion.

Reply:
good callout, done